views/albums.hbs

```diff
@@ -0,0 +1,5 @@
+<h1>Albums</h1>
+
+{{#each albumartist}}
+{{> albumCard this}}
```

Comment:
It is OK to use Partials here but usually they come in handy when the same code is reused in different hbs files.